tools/pd-ctl/pdctl/command/store_command.go

```diff
@@ -243,18 +244,50 @@ func storeLimitSceneCommandFunc(cmd *cobra.Command, args []string) {
 	}
 }
 
+func convertToStoreInfo(content string) string {
+	store := &api.StoreInfo{}
+	err := json.Unmarshal([]byte(content), store)
+	if err != nil {
+		return content
+	}
+	store.Store.State = 0
+	byteArr, err := json.MarshalIndent(store, "", "  ")
+	if err != nil {
+		return content
+	}
+	return string(byteArr)
+}
+
+func convertToStoresInfo(content string) string {
+	stores := &api.StoresInfo{}
+	err := json.Unmarshal([]byte(content), stores)
+	if err != nil {
+		return content
+	}
+	for _, store := range stores.Stores {
+		store.Store.State = 0
+	}
+	byteArr, err := json.MarshalIndent(stores, "", "  ")
+	if err != nil {
+		return content
+	}
+	return string(byteArr)
+}
+
 func showStoreCommandFunc(cmd *cobra.Command, args []string) {
 	prefix := storesPrefix
 	if len(args) > 1 {
 		cmd.Usage()
 		return
 	}
+	cFunc := convertToStoresInfo
 	if len(args) == 1 {
 		if _, err := strconv.Atoi(args[0]); err != nil {
 			cmd.Println("store_id should be a number")
 			return
 		}
 		prefix = fmt.Sprintf(storePrefix, args[0])
+		cFunc = convertToStoreInfo
```

Comment:
Does `else` also need it?

Reply:
Like `prefix`, `cFunc` has assigned before `if` statement